packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -28,6 +28,9 @@ function useAddAppMutation(_type: App["type"] | null, options?: Parameters<typeo
       isOmniInstall = variables.isOmniInstall;
       type = variables.type;
     }
+    if (type === "sendgrid_other_calendar") {
+      type = "sendgrid";
+    }
```

Comment:
Are we losing type inference with this?

Reply:
Not sure of what you mean @zomars. Maybe @hariombalhara can pitch in as he was the one suggesting going this way.

Reply:
I'm bringing this up because of type errors: 
<img width="693" alt="image" src="https://user-images.githubusercontent.com/3504472/202549105-11f6a291-d2fc-44bc-86a2-3b46cd2f97bf.png">